### ISSUE TITLE
3D map issues

### ISSUE THREAD

**Issue by SirBraneDamuj** [SirBraneDamuj/mustadio#12]
I'm about to release a new feature that allows you to see a 3D preview of the map.

A lot of the maps' textures are messed up. I am using a tool that I did not write to generate these models, so this is unfortunately the way of things for now.

However, some of these _are_ fixable, but require some manual intervention from me. TLDR, the script I use produces a lot of variations of the same model. Some of the variations look better than others.

Please leave a comment here with any maps you see that are messed up, and I'll see if there's a better one available.

**Comment by Zagorsek** [comment]
16
82
Also apparently dark mode makes the map modal unreadable, oops my bad

**Comment by SirBraneDamuj** [comment]
>Also apparently dark mode makes the map modal unreadable, oops my bad

Nope, that one's my bad haha. I completely forgot about dark mode.

**Comment by Zagorsek** [comment]
86

**Comment by codfishcartographer** [comment]
62 is super garishly bright red
